README.md

```diff
@@ -70,18 +70,18 @@ $ echo 'eval `luarocks path`' >> ~/.bashrc
 ```
 For further information, consult the [Luarocks documentation](https://github.com/luarocks/luarocks/wiki/path).
 
-### Compiling the runtime libraries
+### Compiling the custom interpreter
 
-Pallene must be run against a custom-built version of the Lua interpreter, as
-well as the Pallene runtime library. Both of these are written in C and must be
-compiled before the Pallene compiler can be used.
+Pallene must be run against a custom-built version of the Lua interpreter.
+This custom version of Lua 5.4 doesn't have to be the same one that you will use to run the compiler itself,
+or to install the Luarocks packages.
 
-These two components can be built through the Makefile we provide. The command
-to be used depends on your operating system:
+To compile the custom version of Lua, follow the instructions found the vm/doc/readme.html.
```

Comment:
> instructions found the

"Instructions found in the [path]" perhaps?

Reply:
Apparently, Github displays the raw HTML if we link to that. I think I'm going to link to the lua.org website instead.